packages/components/src/internal/components/container/FolderAPIWrapper.ts

```diff
@@ -215,6 +220,27 @@ export class ServerFolderAPIWrapper implements FolderAPIWrapper {
     };
 
     getFolderDataTypeExclusions = getFolderDataTypeExclusions;
+
+    updateProjectCustomLabels = (
```

Comment:
It feels a bit odd to prefix this with `Project` when the API is just called `customLabels`

Reply:
good point. I created a LabelsAPIWrapper instead.